tests/components/switch/test_command_line.py

```diff
@@ -162,16 +162,23 @@ def test_assumed_state_should_be_true_if_command_state_is_false(self):
         """Test with state value."""
         self.hass = get_test_home_assistant()
 
-        # Set state command to false
-        statecmd = False
-
-        no_state_device = command_line.CommandSwitch(self.hass, "Test", "echo",
-                                                     "echo", statecmd, None)
+        # args: hass, device_name, friendly_name, command_on, command_off,
+        #       command_state, value_template
+        init_args = [
+                self.hass,
+                "test_device_name",
+                "Test friendly name!",
+                "echo 'on command'",
+                "echo 'off command'",
+                False,
+                None
+        ]
+
+        no_state_device = command_line.CommandSwitch(*init_args)
```

Comment:
Good point, thanks. Would it generally be better for me to change 11779cb to use e.g. `ENTITY_ID_FORMAT.format('test_entity_id')` instead of just `switch.test_entity_id` like I've done?

Also, I should probably add a test verifying that `.name` defaults to the `entity_id` if `friendly_name` is not given, though that logic is in the config reading / setup portion and not in the switch initialization, which means I should either rewrite the test to make the device from an example config (instead of directly from `CommandSwitch`), or perhaps move that logic from the config to the switch initialization (i.e. change `friendly_name` to a kwarg with default `None`). Does any of that stand out as being obviously the best route?